### PR TITLE
OX courier package reflavor

### PR DESCRIPTION
**Pull request by DreamySkrell**
image
image

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!review

File: code/modules/cargo/delivery/package.dm [hunk 1]
```diff
@@ -1,7 +1,18 @@
 /obj/item/cargo_package
 	name = "cargo package"
-	desc = "An Orion Express cargo package, these packages generally find their way to researchers bunkered up on exoplanets. Always pays an extra 2% tip to the courier."
-	desc_info = "You can deliver this package to a delivery site on an exoplanet to get additional funds for the cargo department's account. An additional 2% is added to your account on delivery. Can be loaded into a cargo pack."
+	desc = "\
```
Comment (FluffyGhoster) [inline code/modules/cargo/delivery/package.dm:3]: No wasted lines

File: code/modules/cargo/delivery/package.dm [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+		An Orion Express cargo package. \
+		Always pays an extra 2% tip to the courier.\
+	"
+	desc_extended = "\
```
Comment (FluffyGhoster) [inline code/modules/cargo/delivery/package.dm:7]: .

File: code/modules/cargo/delivery/package.dm [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
+		This package makes use of the small-scale shipping network of Orion Express. \
+		It is a common sight all over the Spur, where Orion Express services depend on ordinary people and ships picking up and delivering packages for each other, \
+		with Orion Express only delivering to automated stations and other distribution points.\
+	"
+	desc_info = "\
```
Comment (FluffyGhoster) [inline code/modules/cargo/delivery/package.dm:12]: .

File: code/modules/cargo/delivery/package.dm [hunk 4]
```diff
@@ -1,7 +1,18 @@
...
+		You can deliver this package to a cargo delivery point. \
+		An additional 2% is added to your account on delivery, or paid to you directly. Can be loaded into a cargo pack.\
+	"
```
Comment (FluffyGhoster) [inline code/modules/cargo/delivery/package.dm:15]: .

File: code/modules/cargo/delivery/receptacle.dm [hunk 5]
```diff
@@ -2,8 +2,19 @@ var/global/list/all_cargo_receptacles = list()
 
 /obj/structure/cargo_receptacle
 	name = "cargo delivery point"
-	desc = "An Orion Express automated cargo acceptance device. It's linked to a vast underground network of conveyors and de-sterilizers, transporting it to its final destination. These are generally used by researchers who find themselves bunkered alone on an exoplanet."
-	desc_info = "This is a delivery point for Orion Express cargo packages. To finish the delivery, have a cargo package in your hand and click on the delivery point."
+	desc = "\
```
Comment (FluffyGhoster) [inline code/modules/cargo/delivery/receptacle.dm:5]: .

File: code/modules/cargo/delivery/receptacle.dm [hunk 6]
```diff
@@ -2,8 +2,19 @@ var/global/list/all_cargo_receptacles = list()
...
+		An Orion Express automated cargo acceptance device. \
+		It is linked to a vast underground network of conveyors and shelves, storing packages for further transport or retrieval.\
+	"
+	desc_extended = "\
```
Comment (FluffyGhoster) [inline code/modules/cargo/delivery/receptacle.dm:9]: .

File: code/modules/cargo/delivery/receptacle.dm [hunk 7]
```diff
@@ -2,8 +2,19 @@ var/global/list/all_cargo_receptacles = list()
...
+		These are set up by Orion to expand its small-scale shipping network, especially in more remote areas, like outer edges of the Frontier or Coalition. \
+		It is a common sight all over the Spur, however, where Orion Express services depend on ordinary people and ships picking up and delivering packages for each other, \
+		with Orion Express only delivering to automated stations and other distribution points.\
+	"
+	desc_info = "\
```
Comment (FluffyGhoster) [inline code/modules/cargo/delivery/receptacle.dm:14]: .

File: code/modules/cargo/delivery/receptacle.dm [hunk 8]
```diff
@@ -2,8 +2,19 @@ var/global/list/all_cargo_receptacles = list()
...
+		This is a delivery point for Orion Express cargo packages. \
+		To finish the delivery, have a cargo package in your hand and click on the delivery point.\
+	"
```
Comment (FluffyGhoster) [inline code/modules/cargo/delivery/receptacle.dm:17]: .

**Review by Arrow768 — APPROVED** [review]

File: code/modules/cargo/delivery/package.dm (hunk 1 above)
Comment (NonQueueingMatt) [reply code/modules/cargo/delivery/package.dm:3]: well it is a bit odd tbh, would prefer if the first line wasn't just \

Comment (DreamySkrell) [reply code/modules/cargo/delivery/package.dm:3]: I think it's fine, and this is a style I used in many files now.
Like this all the text lines are aligned.

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**